bigquery/google/cloud/bigquery/job.py

```diff
@@ -1910,6 +1910,18 @@ def print_header(self):
     def print_header(self, value):
         self._set_sub_prop("printHeader", value)
 
+    @property
+    def use_avro_logical_types(self):
+        """bool: For loads of Avro data, governs whether Avro logical types are
+        converted to their corresponding BigQuery types(e.g. TIMESTAMP) rather than
```

Comment:
Nit: Missing a space between `types` and `(e.g.`

Reply:
Let's then fix the original docstring, too (in `LoadJobConfig.use_avro_logical_types`).